core/tests/webdriverio_utils/AdminPage.js

```diff
@@ -247,6 +277,18 @@ var AdminPage = function() {
       removeButtonElement, 'Role removal button takes too long to appear.');
   };
 
+  this.addLanguageToCoordinator = async function(name, language) {
```

Comment:
How is this different from `makeUserTranslationCoordinator`? In case they are similar, can we do something to avoid duplicate pattern/code?

Reply:
Done. PTAL again. Thanks.